backend/src/routes/cartRoute.js

```diff
@@ -12,16 +12,9 @@ const __dirname = path.dirname(__filename);
 
 // middleware
 
-cartRouter.use(
-  express.static(path.join(__dirname, "../../../frontend/node_modules"))
-);
-cartRouter.use(
-  express.static(path.join(__dirname, "../../../frontend/public"))
-);
-
 cartRouter.use(bodyParser.json());
 cartRouter.use(bodyParser.urlencoded({ extended: true }));
 
-cartRouter.get("/cart", getCart);
+cartRouter.get("/", getCart);
```

Comment:
Not sure you necessarily need a cart router. Looking at it a cart is product user has selected but not yet purchased. More like a pending sales. But if you think a cart is the best way you can implement it, please go ahead. There are always different ways to solve stuff.